rest/model/task.go

```diff
@@ -327,6 +335,17 @@ func (at *APITask) BuildFromService(t interface{}) error {
 
 // ToService returns a service layer task using the data from the APITask.
 func (ad *APITask) ToService() (interface{}, error) {
+	if *ad.Status == evergreen.TaskKnownIssue {
+		task, err := task.FindOneId(*ad.Id)
+		if err != nil {
+			return "", errors.Wrapf(err, "problem finding task %s", *ad.Id)
+		}
+		if task == nil {
+			return "", errors.Errorf("no task found %s", *ad.Id)
```

Comment:
Can this error message be worded differently? "task '%s' not found"  reads more like a sentence